submissions/AnzhelikaKh/popup_task/index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>PopUp</title>
+</head>
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <div class="logo"><a href="./index.html">
+                <img class = "logo__img" src="./img/Logo1.jpg" alt="Logo"></a>
+            </div>
```

Comment:
A logo should be a link and leads users to the home page

Reply:
I already have this link "<a href="./index.html">" :)